vscode/src/documentation.ts

```diff
@@ -14,7 +14,7 @@ export async function showDocumentationCommand(extensionUri: Uri) {
 
   // Reveal panel and show 'Loading...' for immediate feedback.
   sendMessageToPanel(
-    "documentation", // This is needed to route the message to the proper panel
+    { panelType: "documentation", id: "" }, // This is needed to route the message to the proper panel
```

Comment:
Though `""` technically works, I dislike the use of empty string to mean "none". Making `id` an optional field and simply not passing it in here would make the semantics a bit clearer I think.

Reply:
I've made the id field optional, though it gets filled in as `""` when we actually access the underlying map. Maps have to have strings (or numbers) as their keys and can't have `undefined`.